test/src/tests/generated/additional-properties/pets.go

```diff
@@ -91,6 +100,14 @@ func (client PetsClient) CreateAPInPropertiesResponder(resp *http.Response) (res
 
 // CreateAPInPropertiesWithAPString create a Pet which contains more properties than what is defined.
 func (client PetsClient) CreateAPInPropertiesWithAPString(ctx context.Context, createParameters PetAPInPropertiesWithAPString) (result PetAPInPropertiesWithAPString, err error) {
+	ctx = tracing.StartSpan(ctx, "generated/additional-properties/PetsClient.CreateAPInPropertiesWithAPString")
```

Comment:
Per offline discussion please update codegen to not do any of this work if `tracing.Enabled() == false`.